packages/core/rendering/core-rendering-server-internal/src/bootstrap/bootstrap_renderer.test.ts

```diff
@@ -318,7 +342,7 @@ describe('bootstrapRenderer', () => {
 
       expect(getThemeTagMock).toHaveBeenCalledTimes(1);
       expect(getThemeTagMock).toHaveBeenCalledWith({
-        themeVersion: 'v8',
+        name: 'v8',
```

Comment:
so is v8 still the default theme?

Reply:
Correct, `v8` is the theme tag of what kibana ships with right now, and we call it Amsterdam. Up to this point themes were versioned by kibana majors and not named, that's why we have the `v8` theme (and before that we had the `v7` theme) but that's not what we want to do moving forward.

The updated approach is to differentiate themes by their name resulting in theme tags like `borealislight` and `borealisdark` for the new Borealis theme. For compatibility reasons we decided to keep the theme tags for the current Amsterdam theme the same, so `v8light` and `v8dark`, and that's why that specific mapping of `amsterdam` theme name to `v8` is necessary.